lib/tasks/import.thor

```diff
@@ -97,9 +129,14 @@ class Import
     def connect_transform_import
       timing_log = []
 
-      file_importers = importers.flat_map { |i| i.new(options).file_importers }
+      file_import_classes.each do |file_import_class|
+        file_importer = file_import_class.new(
```

Comment:
This initialization is ugly and messy, as a first step to cleaning it up I'd like to add Ruby required keyword arguments.